.travis.yml

```diff
@@ -11,5 +11,5 @@ before_script:
   - composer install
 
 script:
-  - phpunit --coverage-clover build/logs/clover.xml
+  - vendor/bin/phpunit --coverage-clover build/logs/clover.xml --coverage-text
```

Comment:
Sorry I didn't catch this earlier but please remove `--coverage-text`. The coverage being generated is for use with code-climate which parses the default output format.